fdb-record-layer-lucene/src/test/java/com/apple/foundationdb/record/lucene/FDBLuceneQueryTest.java

```diff
@@ -138,14 +144,14 @@ public static void setup() {
             TextSamples.AETHELRED,
             TextSamples.PARTIAL_ROMEO_AND_JULIET_PROLOGUE,
             TextSamples.FRENCH,
-            TextSamples.ROMEO_AND_JULIET_PROLOGUE,
```

Comment:
We should keep this consistent with the rest, and avoid the static import

Reply:
Sure, done.